sentry-android-core/src/main/java/io/sentry/android/core/internal/util/ScreenshotUtils.java

```diff
@@ -55,20 +61,57 @@ public class ScreenshotUtils {
       final Bitmap bitmap =
           Bitmap.createBitmap(view.getWidth(), view.getHeight(), Bitmap.Config.ARGB_8888);
 
-      final Canvas canvas = new Canvas(bitmap);
-      if (mainThreadChecker.isMainThread()) {
-        view.draw(canvas);
-      } else {
-        final @NotNull CountDownLatch latch = new CountDownLatch(1);
-        activity.runOnUiThread(
-            () -> {
-              try {
-                view.draw(canvas);
+      final @NotNull CountDownLatch latch = new CountDownLatch(1);
+
+      // Use Pixel Copy API on new devices, fallback to canvas rendering on older ones
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+
+        final HandlerThread thread = new HandlerThread("screenshot");
```

Comment:
```suggestion
        final HandlerThread thread = new HandlerThread("SentryScreenshot");
```
to make it more specific to our sdk